### DISCUSSION TITLE
Mod support for /worth, /setworth etc.

### DISCUSSION THREAD

**Discussion by SenZ01** [Zrips/CMI#4609] [Feature Requests]
**Description of feature/improvement request:**

It would be a nice addition to have Modded support for Setworth for Modpack Creators / Server Owners
- - - - - - - - - - - - - - - - - - - - - - - - - - -

### Detailed explanation of what it should do, when and in what situation: ###

What it should do: allow us to set the worth for modded items via numeric:ID or mod:ID

- - - - - - - - - - - - - - - - - - - - - - - - - - -

### Basic explanation as to why this feature is needed and/or useful to have: ###

I've seen many modded Servers that use CMI, many of which don't have any Economy systems besides the usuall Signshops, so it would be amazing to be able to set worth for modded items so you can actually /sell them in just one command without having to trade everything at hand manually with players, or switch to other plugins that could do simmilar things.
- - - - - - - - - - - - - - - - - - - - - - - - - - -

Thank you for taking ur time reading this.
Best regards; SenZ0.

**Comment by CloudeLecaw** [comment]
Dont be disapointed but this will definatly not come so fast,
Support for modded items was wished in a sections allready months (years?) ago like as materials for custom recips. In some sections like kit this was added, in some others it may be cant, i think setworth is one of them.

**Comment by SenZ01** [comment]
> Dont be disapointed but this will definatly not come so fast,
> Support for modded items was wished in a sections allready months (years?) ago like as materials for custom recips. In some sections like kit this was added, in some others it may be cant, i think setworth is one of them.

I can hope and dream that it will be possible one day :D!
